### PR TITLE
Reviewed: 1204-recipient-country-budget-status-field-missing-in-aidstream

### PR DESCRIPTION
**Pull request by PG-Momik**
Set Recipient country budget status to indicative when migrating + fix migrated organization's Recipient country budget status

### CODE REVIEW

File: app/Console/Commands/FixIndicative.php [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+use App\IATI\Traits\MigrateGeneralTrait;
+use App\IATI\Traits\MigrateOrganizationTrait;
+use Illuminate\Console\Command;
+use Illuminate\Database\DatabaseManager;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Validator;
+use Throwable;
+use function PHPUnit\Framework\isNull;
+
+class FixIndicative extends Command
```
Comment (Sanilblank) [inline app/Console/Commands/FixIndicative.php:18]: doc comment

File: app/Console/Commands/FixIndicative.php [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    protected $signature = 'command:FixIndicative';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Adds indicative to recipient country budget >> status for migrated organizations';
+
+    public function __construct(
```
Comment (Sanilblank) [inline app/Console/Commands/FixIndicative.php:38]: doc comment

File: app/Console/Commands/FixIndicative.php [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    }
+
+    /**
+     * Returns array of [organizationId => organizationIdentifier].
+     *
+     * @param $aidstreamOrganizationIds
+     *
+     * @return array
+     */
+    private function getAidstreamOrganizationIdentifier($aidstreamOrganizationIds): array
```
Comment (Sanilblank) [inline app/Console/Commands/FixIndicative.php:149]: may need to take organization identifier from organization table if settings registry info does not exist

File: app/Console/Commands/FixTransaction.php [hunk 4]
```diff
@@ -44,12 +44,12 @@ class FixTransaction extends Command
      * @param ActivityService $activityService
      * @param OrganizationService $organizationService
      */
-    public function __construct(protected DB $db,
+    public function __construct(
```
Comment (Sanilblank) [inline app/Console/Commands/FixTransaction.php:47]: doc comment

**Review by sriza — APPROVED** [review]

**Pull request merged**